Tasks/Xcode/task.json

```diff
@@ -12,7 +12,7 @@
     "version": {
         "Major": 5,
         "Minor": 133,
-        "Patch": 0
+        "Patch": 1
```

Comment:
No need to bump the patch version since 5.133 hasn't shipped.

Reply:
It doesn't have to change since 5.133 hasn't shipped, but after Jason's comment I think it's a good practice to always* bump the version number. It's a simple rule to follow, and if we're good about bumping the version then it's easier to correlate a build log to our source tree at a particular commit.

*With a possible exception for widespread non-critical changes to several tasks at once (e.g. a find-replace to fix the casing of "GitHub").